pandas-iris/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/pipeline_registry.py

```diff
@@ -3,22 +3,19 @@
 
 from kedro.pipeline import Pipeline
 
-from {{cookiecutter.python_package}}.pipelines import data_engineering as de
-from {{cookiecutter.python_package}}.pipelines import data_science as ds
+from {{ cookiecutter.python_package }} import pipeline as mp
 
 
 def register_pipelines() -> Dict[str, Pipeline]:
     """Register the project's pipelines.
 
     Returns:
         A mapping from a pipeline name to a ``Pipeline`` object.
-
     """
-    data_engineering_pipeline = de.create_pipeline()
-    data_science_pipeline = ds.create_pipeline()
+
+    my_pipeline = mp.create_pipeline()
```

Comment:
```suggestion
```

Reply:
Ah no sorry, I messed up the suggestions... I meant no need to define `my_pipeline` at all. You can just do `"default": create_pipeline()` directly.